raft/log.go

```diff
@@ -40,7 +40,7 @@ type raftLog struct {
 	logger Logger
 
 	// maxNextEntsSize is the maximum number aggregate byte size of the messages
-	// returned from calls to nextEnts.
+	// returned from calls to nextCommittedEnts.
 	maxNextEntsSize uint64
```

Comment:
minor comment: Probably it makes sense to rename `maxNextEntsSize` to `maxNextCommittedEntsSize`.

Reply:
Good idea. Done.